src/pages/components/table.mdx

```diff
@@ -165,6 +165,28 @@ import TruncateCode from '!!raw-loader!../../examples/table/TableTruncate.tsx';
   <Truncate />
 </CodeExample>
 
+### Draggable
+
+This functionality is implemented using [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd).
+
+import TableDraggable from '../../examples/table/TableDraggable.tsx';
+import TableDraggableCode from '!!raw-loader!../../examples/table/TableDraggable.tsx';
+
+<CodeExample code={TableDraggableCode}>
+  <TableDraggable />
+</CodeExample>
+
+### Virtual scrolling
+
+Virtualized tables help efficiently render large amounts of data. This functionality is implemented using [react-window](https://react-window.now.sh/#/examples/list/fixed-size).
```

Comment:
Recommend rewriting

> Virtualized tables help efficiently render large amounts of data.

to "Efficiently render large amounts of Table data." I'm not entirely convinced the "functionality is implemented using..." sentence is necessary. The implementation makes this clear at a glance.

Reply:
Updated for the more compact descriptive.

My vote is to keep as much of the implementation details as possible for dragging and virtualization. With products having to roll this functionality themselves I've seen designers use this info as well.